test/test_save_as_netcdf.py

```diff
@@ -30,7 +30,6 @@ def nc_has_variable(ds, var_name):
     if var_name in ds.variables:
         return True
     else:
-        print(var_name, " is not a variable in the Dataset")
```

Comment:
These prints may be worth keeping -- if a test fails, you may not know which variable, attribute, etc is causing the failure, so this is kind of handy.